client_test/rpc_test.go

```diff
@@ -0,0 +1,61 @@
+package client_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/rs/zerolog"
+	"github.com/statechannels/go-nitro/client/engine/chainservice"
+	p2pms "github.com/statechannels/go-nitro/client/engine/messageservice/p2p-message-service"
+	"github.com/statechannels/go-nitro/client/rpc"
+	"github.com/statechannels/go-nitro/internal/testactors"
+	"github.com/statechannels/go-nitro/internal/testdata"
+	"github.com/statechannels/go-nitro/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRpcClient(t *testing.T) {
```

Comment:
Do you think this test should replace the [rpcnats_test](https://github.com/statechannels/go-nitro/blob/2da58eaed4180f36b1d28cfaf89595307471f627/client_test/rpcnats_test.go#L30)?

Reply:
Yup I think so

Reply:
Excellent!